docs/how-to/installation.md

```diff
@@ -34,3 +34,18 @@ The locales that are synchronised from others can be managed through the Wagtail
 
 To enable this, replace the ``"wagtail.locales"`` entry in ``INSTALLED_APPS`` with ``"wagtail_localize.locales"``, this
 will add a "Sync from" field to all locales that allows an administrator to choose a locale to synchronise content from.
+
+
+## Disabling the default translation mode
+
+Wagtail Localize will automatically go in translation mode when creating new translations for models.
+
+To disable this behaviour globally, set `WAGTAIL_LOCALIZE_DISABLE_TRANSLATION_MODE = True` in your settings file. 
```

Comment:
Wondering if "disable" is the best wording here? As it doesn't prevent the user from using any translation modes

How about ``WAGTAIL_LOCALIZE_DEFAULT_TRANSLATION_MODE = 'synced' / 'simple'``?

Reply:
I went in circles a few times. Happy to change this